src/jira/util/jwt.ts

```diff
@@ -75,7 +75,7 @@ function decodeAsymmetricToken(token: string, publicKey: string, noVerify: boole
 
 
 function verifyQsh(qsh: string, req: Request): boolean {
-	const requestInAtlassianJwtFormat = {...req, pathname: req.path}
```

Comment:
This completely broke because `req.path` is not the correct variable to use as it's only the router specific path, not the full path.